src/components/avatar/utils.ts

```diff
@@ -63,3 +78,39 @@ export function rgbToHue(rgb: RGB): number {
 export function hexToHue(hex: string): number {
   return rgbToHue(hexToRGB(hex));
 }
+
+/**
+ * For RGB color, find luminance
+ *
+ * @param colorObject {RGB}
+ */
+export function rgbToLuminosity(colorObject: RGB): number {
+  const color = Color(colorObject);
+  const luminosity = color.luminosity();
+  return luminosity;
+}
+
+/**
+ * Verify the generated color is WCAG contrast compliant
+ *
+ * @param hex {string} - form of "#------"
+ * @param theme {string}
+ */
+export function isContrastCompliant(hex: string, theme: string): string | null {
+  const getLuminance = (hex) => {
+    const rgbObj = hexToRGB(hex);
```

Comment:
This isn't needed as `color` can deal with different color models (i.e., you should be able to get luminosity from a hex-created color instance).